kansha/card_addons/gallery/comp.py

```diff
@@ -79,11 +117,12 @@ def add_asset(self, new_file):
             - The newly created Asset
         """
         validators.validate_file(new_file, self.assets_manager.max_size, _(u'File must be less than %d KB'))
```

Comment:
Move this `validate_file` validator out of the `kasha.authentication.database` module